### ISSUE TITLE
Work in simulator not work in iPhone device

### ISSUE THREAD

**Issue by zixun** [isagalaev/highlight.js#922]
dont know why

**Comment by isagalaev** [comment]
It's impossible to do anything from this description. Please do some investigation to show what is exactly wrong with highlight.js here.

**Comment by Sannis** [comment]
Looks like a duplicate of #891. My apologises, but we need help from experienced mobile web developers to investigate this.

**Comment by zixun** [comment]
@Sannis  @isagalaev  i wrote a demo for this [TestHighLight](https://github.com/zixun/TestHighLight), highlight.js is work in simulator not work in iPhone device, please use Xcode7-Bete6,it can use device testing without Certificate。please inform me If there is any research results。 I'm using highlight.js and my application is about to go online. thank you~

**Comment by isagalaev** [comment]
@zixun what you're proposing is that some of use should get a Mac, set it up, buy or license a particular development environment, learn it, learn how to build your project, get an iPhone, figure out how to upload your app to it… and then try to guess how exactly you were expecting highlight.js to work and what does it mean that it doesn't. All that to find a problem that most probably has nothing to do with highlight.js itself, as you say it works fine in an emulator.

In other words, you're in a much better position to investigate it. It looks like the iPhone either can't find the script at the path you provide or may be your app doesn't have permissions for it or something like that in an environment.

**Comment by isagalaev** [comment]
btw, quickly looking through your code revealed at least one possible reason: you're misspeled "arta.css" with a capital letter here: https://github.com/zixun/TestHighLight/blob/master/html/test.html#L12

**Comment by zixun** [comment]
@isagalaev  I'm so sorry,it's my fault！highlight.js works well at device while change to 'arta.css'. Thx
